subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/AbstractGradleExecuter.java

```diff
@@ -1061,19 +1086,28 @@ private void validate(String output, String displayName) {
                     if (line.matches(".*use(s)? or override(s)? a deprecated API\\.")) {
                         // A javac warning, ignore
                         i++;
+                    } else if (line.contains("Support for running Gradle using Java 7 has been deprecated and is scheduled to be removed in Gradle 5.0")) {
```

Comment:
Can you explain why we need those changes? Is this to prevent tests to fail because they are now emitting deprecation warnings if run on Java 7-based CI jobs?

Reply:
Yes. All integration tests would be affected on jdk7. There's a deprecation warning check in this class.